src/site/content/en/blog/bfcache/index.md

```diff
@@ -188,9 +188,9 @@ It means that the browser _intends_ to cache the page, but there may be factors
 that make it impossible to cache.
 
 ```js
-window.addEventListener('pagehide', function(event) {
+window.addEventListener('pagehide', (event) => {
   if (event.persisted === true) {
```

Comment:
```suggestion
  if (event.persisted) {
```

Reply:
Oh, sorry, didn't see this until after I pressed merge...

In my normal code I'd probably write it the short way, but I think I originally wanted to make it clear that this was a boolean flag (can't remember now actually).

Reply:
All good! It was a minor style thing anyway. :)